product/models.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+from user.models import User
+
+class MainMenu(models.Model): 
+    email = models.CharField(max_length = 200, help_text = "이메일")
+    class Meta: 
+        db_table = "main_menus"
+
+class ProductCategory(models.Model): 
```

Comment:
```suggestion
class Category(models.Model): 
```

이미 `product` 앱 내부에 있는 클래스이기 때문에 따로 프로덕트로 쓰지 않으셔도 될 것 같습니다. 클래스 이름이 불필요하게 길면 나중에 로직을 구성할 때 한 번에 알아보기 힘들 수 있습니다. 아래 `ProductSubCategory` 도 마찬가지입니다 :)